dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.mpTelemetry1.1.ee-7.0.feature

```diff
@@ -1,6 +1,7 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
 symbolicName=io.openliberty.mpTelemetry1.1.ee-7.0
 singleton=true
+IBM-API-Package: io.openliberty.microprofile.telemetry.api;type="ibm-api"
```

Comment:
I'm not sure why this header is in each private feature. As far as I can see it's the same in all of them? Could it go in the public feature instead?